examples/ai-core/package.json

```diff
@@ -48,5 +49,6 @@
     "@types/node": "20.11.20",
     "tsx": "4.19.2",
     "typescript": "5.6.3"
-  }
+  },
+  "type": "module"
```

Comment:
why is this needed?

Reply:
running the example i added errors without this! one of the newly introduced deps (pkce-challenge) lacks cjs compatibility

Reply:
Fixed by updating build command! `npx tsc src/mcp/server.ts --outDir /dist --target es2023 --module nodenext`